src/types/account.ts

```diff
@@ -74,6 +75,7 @@ export enum AccountSearchType {
   ContractStorage,
   ContractCode,
   InternalTxReceipt,
+  // SecureAccount,
```

Comment:
why is this added as comment?

Reply:
The commented-out SecureAccount type was added to the AccountSearchType enum in account.ts as a placeholder for future secure account search functionality. Removed it in the latest commit